content/02.main-text.md

```diff
@@ -19,6 +19,11 @@ The Deep Review and other studies that subsequently adopted the Manubot platform
 However, inviting wide authorship brought many technical and social challenges such as how to fairly distribute credit, coordinate the scientific content, and collaboratively manage extensive reference lists.
 The manuscript writing process we developed using the Markdown language, the GitHub platform, and our new Manubot tool for automating manuscript generation addresses these challenges.
 
+Manubot supports citations by adding a standard identifier like a Digital Object Identifier (DOI) or PubMed Identifier (PMID) directly in the text so that large groups of authors do not have to coordinate reference lists.
+When text is changed, Manubot automatically updates the manuscript's web page so that all authors can read and edit from the latest version.
+Because Manubot manuscripts are created from GitHub repositories, all edits are reviewed and discussed, ensuring that the collaborative text has a cohesive style and message and that authors receive precise credit for their work.
```

Comment:
> Because Manubot manuscripts are created from GitHub repositories, all edits are reviewed and discussed

What about users who commit directly to the master branch and don't use PRs? IMO Manubot allows a lot of flexibility in terms of how commits are added to the `master` branch. For some projects, it makes sense to use PRs. For others, it may not.

Reply:
That is certainly possible, but it isn't the contribution workflow we describe in the paper or recommend.  I'd like to advertise the main reasons to use Manubot here, and I think this is one of them.  Collaborative writing without a review system can get confusing to track as the number of authors grows.

Is there a better and more accurate way to phrase this?  Or should we add the detail in the contribution workflow section that other workflows are possible and this is a project-specific choice?

Reply:
What about:

> Because manuscripts are created from GitHub repositories, Manubot supports a workflow where all edits are reviewed and discussedd, ensuring that the collaborative text has a cohesive style and message and that authors receive precise credit for their work.

Reply:
> should we add the detail in the contribution workflow section that other workflows are possible and this is a project-specific choice?

I think a short note on this topic is also a good idea. I'd venture that perhaps close to 50% of the manubot manuscripts in existence do not use a PR model and instead primarily commit directly to their master branch. In fact, some don't even ever push the repo to GitHub it seems.

Perhaps we could add a paragraph like:

***
While we developed Manubot with collaborative writing in mind, it can also be helpful for individuals preparing scholarly documents. In some cases, authors may choose to make their changes directly to the `master` branch, forgoing pull requests and the associated review. This workflow retains many of the benefits, such as transparent history, automation, and allowing proposed changes from outside contributors. In cases where outside contributions are unwanted, authors can disable pull requests on GitHub.
***
@agitter feel free to take any parts of that (or I can open a separate PR)?

Reply:
I like both of you quoted text blocks above.  I can work them into this PR tonight.

You're right that we need to clarify that Manubot can be used in many different modes and is even useful for single authors.